types/k6/experimental/browser.d.ts

```diff
@@ -1,3 +1,5 @@
+export {};
```

Comment:
This helps to enforce us to be explicit in what we export, otherwise all types are exported

Reply:
TIL 🙇🏻